### PR TITLE
remove contrib govcloud

### PR DESCRIPTION
**Pull request by enxebre**

### CODE REVIEW

File: tests/rspec/lib/config_file.rb [hunk 1]
```diff
@@ -22,7 +22,7 @@ def networking
   end
 
   def node_count
-    master_count + worker_count
+    master_count + worker_count + etcd_count
```
Comment (squat) [inline tests/rspec/lib/config_file.rb]: This should go in another PR, no?

Comment (enxebre) [reply tests/rspec/lib/config_file.rb]: sure https://github.com/coreos/tectonic-installer/pull/3287, lets see if we get it green asap. This is due to tests were not run here https://github.com/coreos/tectonic-installer/pull/3285

**Comment by enxebre** [comment]
retest this please

**Comment by enxebre** [comment]
retest this please
